rackcity/views/it_instance_views.py

```diff
@@ -459,3 +459,21 @@ def instance_page_count(request):
     instance_count = ITInstance.objects.all().count()
     page_count = math.ceil(instance_count / page_size)
     return JsonResponse({"page_count": page_count})
+
+
+@api_view(['GET'])
+def instance_fields(request):
+    """
+    Return all fields on the ITInstanceSerializer. 
+    """
+    return JsonResponse(
+        {"fields": [
+            'hostname',
+            'model',
+            'rack',
+            'elevation',
+            'owner',
+            'comment',
+        ]},
```

Comment:
Is it hard to get the model or serializer to give you the fields dynamically?  I vaguely remember trying to do it on a model and you could but the response was ugly, but maybe serializer would be better since it contains a list of fields as a property?

I'm also fine with leaving this for now and refactoring later if we feel like it

Reply:
Not sure if this is relevant, but I requested that we return the fields in a certain order (having elevation come right after rack) since the front end dynamically loads in these fields.